### PR TITLE
updated lya config files after changes in softening and winds

### PR DESCRIPTION
**Pull request by andreufont**
I modified the configuration files in examples/lya and examples/dm-only to work well with the new softening settings and WindOn option.

Also, I commented out the BlackHole options from the configuration files, although I'm not sure this is the right thing to do... Do you expect most people to compile the code with the BlackHole option? 

PS: this is also my first attempt to merge from a fork, I hope I don't break the whole internet.

### CODE REVIEW

File: examples/lya/paramfile.gadget [hunk 1]
```diff
@@ -63,9 +52,9 @@ QuickLymanAlphaProbability = 1 # Set to 1.0 to turn dense gas directly into star
 
 SnapshotWithFOF = 1
 WindModel = nowind
-BlackHoleOn = 0
-BlackHoleFeedbackFactor = 0.05
-BlackHoleFeedbackRadius = 0.
-BlackHoleFeedbackRadiusMaxPhys = 0.
-BlackHoleFeedbackMethod = spline | mass
-SeedBlackHoleMass = 5.0e-5
+# if compiled with BlackHole option, add these lines
```
Comment (rainwoodman) [inline examples/lya/paramfile.gadget]: I suppose these blackhole stuff are all optional now too... Do you want to get rid of them while you are still on it?

**Comment by andreufont** [comment]
I've added a new commit where I remove the BlackHole options alltogether.

File: examples/dm-only/paramfile.gadget [hunk 2]
```diff
@@ -29,7 +24,7 @@ StarformationOn = 0
 StarformationCriterion = density
 RadiationOn = 0
 HydroOn = 0
-BlackHoleOn = 0
```
Comment (sbird) [inline examples/dm-only/paramfile.gadget]: I think BlackHoleOn is still required, isn't it? Otherwise looks good.

**Comment by sbird** [comment]
I want to merge this so I can rebase the neutrino branch. I will restore the BlackHoleOn parameters afterwards.

**Comment by sbird** [comment]
I see that BlackHoleOn is required only when BLACK_HOLES is enabled: we should probably change the code so that the same parameters are needed irrespective of the compile-time options (they would just do nothing if the compile option was not enabled). We can do that later though.

**Comment by rainwoodman** [comment]
Yes. I'll merge this one now.

**Comment by andreufont** [comment]
Yes, as Simeon says BlackHoleOn = 0 is required if you compile the code with BlackHole option, but if you did not, then it would crash if you have the option. It would be good to modify the code so that it doesn't crash in any case. I'll start an issue.

**Comment by sbird** [comment]
No worries, I'll fix it now.

**Pull request merged**